### PR TITLE
Cleanup Oregon City

### PR DESCRIPTION
**Pull request by natesales**

### CODE REVIEW

File: src/main/java/org/team1540/robot2022/commands/intake/Intake.java [hunk 1]
```diff
@@ -43,14 +43,27 @@ public void setFold(boolean isUp) {
         }
     }
 
+    /**
+     * Set intake roller percent output
+     *
+     * @param percent intake roller percent output
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/commands/intake/Intake.java:49]:
```suggestion
     * @param intake roller percent output
```

File: src/main/java/org/team1540/robot2022/commands/shooter/Shooter.java [hunk 2]
```diff
@@ -94,17 +99,20 @@ public void setVelocityRPM(TalonFX motor, double velocity) {
...
      *
-     * @param motor    to set
-     * @param velocity to set in RPM
+     * @param frontVelocity front wheel RPM setpoint
+     * @param rearVelocity  front wheel RPM setpoint
      */
     public void setVelocityRPM(double frontVelocity, double rearVelocity) {
         shooterMotorFront.set(TalonFXControlMode.Velocity, (frontVelocity * 2048.0) / 600);
-        shooterMotorRear.set(TalonFXControlMode.Velocity, (rearVelocity* 2048.0) / 600);
+        shooterMotorRear.set(TalonFXControlMode.Velocity, (rearVelocity * 2048.0) / 600);
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/commands/shooter/Shooter.java:110]: We might want to store `2048.0/600` as a constant somewhere

Comment (natesales) [reply src/main/java/org/team1540/robot2022/commands/shooter/Shooter.java]: Good point. I've added the getter/setter methods to ChickenTalonFX. [358200d](https://github.com/flamingchickens1540/toothless/pull/106/commits/358200d2097a0c62c2f73676c57d0794f7e2413f)

File: src/main/java/org/team1540/robot2022/commands/shooter/Shooter.java [hunk 3]
```diff
@@ -56,55 +55,53 @@ public Shooter() {
...
-    }
-
     /**
      * Set motor velocity
      *
      * @param motor    to set
      * @param velocity to set in RPM
      */
-    public void setVelocityRPM(TalonFX motor, double velocity) {
+    public void setVelocityRPM(ChickenTalonFX motor, double velocity) {
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/commands/shooter/Shooter.java:82]: This method feels redundant. If you already have the motor to pass into the function, why not call motor.setVelocityRPM() directly?

File: src/main/java/org/team1540/robot2022/commands/shooter/ShootSequence.java [hunk 4]
```diff
@@ -70,8 +70,8 @@ public ShootSequence(Shooter shooter, Indexer indexer, Drivetrain drivetrain, Ho
                     // rearVelocity = SmartDashboard.getNumber("shooter/tuning/rearRPM", 0);
 
                     hood.set(hoodState);
-                    shooter.setVelocityRPM(shooter.shooterMotorFront, frontVelocity);
-                    shooter.setVelocityRPM(shooter.shooterMotorRear, rearVelocity);
+                    shooter.shooterMotorFront.setVelocityRPM(frontVelocity);
+                    shooter.shooterMotorRear.setVelocityRPM(rearVelocity);
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/commands/shooter/ShootSequence.java:74]:
```suggestion
                    shooter.setVelocityRPM(frontVelocity, rearVelocity);
```

**Review by rutmanz — APPROVED** [review]

**Pull request merged**